src/isomorphic/classic/element/ReactCurrentOwner.js

```diff
@@ -11,20 +11,6 @@
 
 'use strict';
 
-/**
- * Keeps track of the current owner.
- *
- * The current owner is the component who should own any components that are
- * currently being constructed.
- */
-var ReactCurrentOwner = {
-
-  /**
-   * @internal
-   * @type {ReactComponent}
-   */
-  current: null,
-
-};
+var ReactCurrentOwner = require('react-current-owner');
```

Comment:
You can't, unless you want to come work at FB :) Introducing a normal node require into `@providesModule` code requires more work and doing things in a different part of the codebase that just means we have a little bit more to keep in mind when we sync.